Makefile

```diff
@@ -141,17 +140,12 @@ install-service-systemd:
 	install -Dm644 dunst.systemd.service ${DESTDIR}${SERVICEDIR_SYSTEMD}/dunst.service
 endif
 
-uninstall: uninstall-service
-	rm -f ${DESTDIR}${PREFIX}/bin/dunst
-	rm -f ${DESTDIR}${MANPREFIX}/man1/dunst.1
-	rm -rf ${DESTDIR}${PREFIX}/share/dunst
-
-uninstall-service: uninstall-service-dbus
-uninstall-service-dbus:
-	rm -f ${DESTDIR}${SERVICEDIR_DBUS}/org.knopwob.dunst.service
-
-ifneq (0,${SYSTEMD})
-uninstall-service: uninstall-service-systemd
-uninstall-service-systemd:
-	rm -f ${DESTDIR}${SERVICEDIR_SYSTEMD}/dunst.service
-endif
+uninstall:
+	( \
+		rm -f ${DESTDIR}${PREFIX}/bin/dunst || ec=1; \
+		rm -f ${DESTDIR}${MANPREFIX}/man1/dunst.1 || ec=1; \
+		rm -rf ${DESTDIR}${PREFIX}/share/dunst || ec=1; \
+		rm -f ${DESTDIR}${SERVICEDIR_DBUS}/org.knopwob.dunst.service || ec=1; \
+		if [ "0" != "${SYSTEMD}" ]; then rm -f ${DESTDIR}${SERVICEDIR_SYSTEMD}/dunst.service || ec=1; fi; \
```

Comment:
That wasn't exactly what I meant, more like a "|| echo "WARNING: Failed to remove the {"systemd service file", "dbus service file"}"